libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -35,6 +35,29 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::frontend;
 
+string YulUtilFunctions::convertExternalFunction(const Type& _from, const Type& _to)
```

Comment:
Added a conversion function for combined address and selector. The function simply forwards the input. The only advantage is the `solAssert`.